<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseWidget</class>
 <widget class="QWidget" name="DatabaseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(194, 194, 194);</string>
       </property>
       <property name="text">
        <string>List of groups:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="listView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>150</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QListView {
           border-style: none;
           background-color: rgb(39, 39, 39);
		   color: rgb(194, 194, 194);
        }

        QListView::item {
            background-color: rgb(39, 39, 39);
           	border-bottom: 1px solid #DDDDDD;
			color: rgb(194, 194, 194);
			padding-top: 3px;
			padding-bottom: 3px;
        }
        QListView::item:selected {
           background-color: rgb(0, 85, 0);
        }

QListView:focus {
           outline: 1px solid transparent;
       }</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(194, 194, 194);</string>
       </property>
       <property name="text">
        <string>Filters:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_all">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	background-color: rgb(40, 40, 43);
	color: rgb(194, 194, 194);
	border-style: solid;
	border-width: 2px;
	border-color: rgb(59, 59, 59);
	border-radius: 3px;
}

QPushButton::hover
{
	border-color: rgb(38, 38, 38);
}

QPushButton::pressed
{
	background-color: rgb(26, 26, 26);
	border: none;
}</string>
       </property>
       <property name="text">
        <string>All entries</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RepeatedPasswordButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	background-color: rgb(40, 40, 43);
	color: rgb(194, 194, 194);
	border-style: solid;
	border-width: 2px;
	border-color: rgb(59, 59, 59);
	border-radius: 3px;
}

QPushButton::hover
{
	border-color: rgb(38, 38, 38);
}

QPushButton::pressed
{
	background-color: rgb(26, 26, 26);
	border: none;
}</string>
       </property>
       <property name="text">
        <string>Repeated passwords</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_weak">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	background-color: rgb(40, 40, 43);
	color: rgb(194, 194, 194);
	border-style: solid;
	border-width: 2px;
	border-color: rgb(59, 59, 59);
	border-radius: 3px;
}

QPushButton::hover
{
	border-color: rgb(38, 38, 38);
}

QPushButton::pressed
{
	background-color: rgb(26, 26, 26);
	border: none;
}</string>
       </property>
       <property name="text">
        <string>Weak passwords</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTableView" name="tableView">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>194</red>
             <green>194</green>
             <blue>194</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>39</red>
             <green>39</green>
             <blue>39</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>194</red>
             <green>194</green>
             <blue>194</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>194</red>
             <green>194</green>
             <blue>194</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>39</red>
             <green>39</green>
             <blue>39</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>39</red>
             <green>39</green>
             <blue>39</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="PlaceholderText">
           <brush brushstyle="SolidPattern">
            <color alpha="128">
             <red>194</red>
             <green>194</green>
             <blue>194</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>194</red>
             <green>194</green>
             <blue>194</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>39</red>
             <green>39</green>
             <blue>39</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>194</red>
             <green>194</green>
             <blue>194</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>194</red>
             <green>194</green>
             <blue>194</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>39</red>
             <green>39</green>
             <blue>39</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>39</red>
             <green>39</green>
             <blue>39</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="PlaceholderText">
           <brush brushstyle="SolidPattern">
            <color alpha="128">
             <red>194</red>
             <green>194</green>
             <blue>194</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>194</red>
             <green>194</green>
             <blue>194</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>39</red>
             <green>39</green>
             <blue>39</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>194</red>
             <green>194</green>
             <blue>194</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>194</red>
             <green>194</green>
             <blue>194</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>39</red>
             <green>39</green>
             <blue>39</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>39</red>
             <green>39</green>
             <blue>39</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="PlaceholderText">
           <brush brushstyle="SolidPattern">
            <color alpha="128">
             <red>194</red>
             <green>194</green>
             <blue>194</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableView
{
background-color: rgb(39, 39, 39);
color: rgb(194, 194, 194);
border-style: none;
}
QHeaderView::up-arrow { image: url(:/Icons/up-arrow-angle.png); subcontrol-position: right;}

QHeaderView::down-arrow { image: url(:/Icons/down-arrow.png); subcontrol-position: right;}

QHeaderView::section 
{
background-color: rgba(45, 45, 45, 0.5); 
font-size: 18px;
color: rgb(194, 194, 194);
}</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="gpup_name_label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(194, 194, 194);</string>
       </property>
       <property name="text">
        <string>   Root / facebook</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true">QGroupBox
{
	 background-color: rgb(39, 39, 39);
	 border-width: 1px;
     border-style: solid;
     border-color: rgb(102, 102, 102);
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="username_label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(194, 194, 194);
background-color: rgb(39, 39, 39);</string>
              </property>
              <property name="text">
               <string>Username: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="username_label_2">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(194, 194, 194);
background-color: rgb(39, 39, 39);</string>
              </property>
              <property name="text">
               <string>Username: </string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="password_label">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(194, 194, 194);
background-color: rgb(39, 39, 39);</string>
              </property>
              <property name="text">
               <string>Password:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="password_lineEdit">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>ArrowCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit
{
	background-color: rgb(39, 39, 39);
    border-style: none;
	color: rgb(194, 194, 194);
}</string>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pass_button">
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	background-color: rgb(40, 40, 43);
	color: rgb(194, 194, 194);
	border-style: none;
	border-width: 0px;
}

QPushButton::pressed
{
	background-color: rgb(26, 26, 26);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="notes_label">
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(194, 194, 194);
background-color: rgb(39, 39, 39);</string>
              </property>
              <property name="text">
               <string>Notes:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="notes_label_2">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(194, 194, 194);
background-color: rgb(39, 39, 39);</string>
              </property>
              <property name="text">
               <string>Notes:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="url_label">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(194, 194, 194);
background-color: rgb(39, 39, 39);</string>
              </property>
              <property name="text">
               <string>Url:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="url_label_2">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(194, 194, 194);
background-color: rgb(39, 39, 39);</string>
              </property>
              <property name="text">
               <string>Url:</string>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="created_label">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(194, 194, 194);
background-color: rgb(39, 39, 39);</string>
              </property>
              <property name="text">
               <string>Created:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="created_label_2">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(194, 194, 194);
background-color: rgb(39, 39, 39);</string>
              </property>
              <property name="text">
               <string>Created:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="modified_label">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(194, 194, 194);
background-color: rgb(39, 39, 39);</string>
              </property>
              <property name="text">
               <string>Modified</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="modified_label_2">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(194, 194, 194);
background-color: rgb(39, 39, 39);</string>
              </property>
              <property name="text">
               <string>Modified</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
